.travis.yml

```diff
@@ -16,17 +16,31 @@ before_script:
 
 notifications:
   email:
-    - malarzm@gmail.com
+    - walid.asbaiti@gmail.com
```

Comment:
You can remove `notifications` section all together, I did so in https://github.com/codete/FormGeneratorBundle/commit/0cf28fbfe902721543be4510b8dc5125b94d5384